README.md

```diff
@@ -189,11 +191,11 @@ Task/Handler: using git module
 
 [ANSIBLE0002] Trailing whitespace
 examples/example.yml:13
-    action: do nothing   
```

Comment:
Did you remove the trailing whitespace from trailing whitespace examples?

Reply:
oh, good spot.